sagemaker-pipeline-multi-model/restate-project.ipynb

```diff
@@ -2,31 +2,31 @@
  "cells": [
```

Comment:
<p>Why does the endpoint name need to be changed? What should it be changed to?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
When a model is deployed using [MLOps template for model deployment](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-projects-templates-sm.html#sagemaker-projects-templates-code-commit), the endpoint name has format <SageMaker project name>-staging. I have updated the notebook to use the same endpoint name and removed the instruction to update the endpoint to avoid confusion. Please see change in commit https://github.com/aws/amazon-sagemaker-examples/pull/3488/commits/1b9591440411af0a05d454ed9219d44d94ec3da5. Thank you!